src/test/java/uk/gov/hmcts/et/common/model/bulk/types/DynamicFixedListTypeTest.java

```diff
@@ -13,25 +17,23 @@ public class DynamicFixedListTypeTest {
 
     @Test
     public void testGetSelectedLabel() {
-        var dynamicFixedListType = create(null);
+        DynamicFixedListType dynamicFixedListType = create(null);
         assertNull(dynamicFixedListType.getSelectedLabel());
-
         dynamicFixedListType = create(new DynamicValueType("code1", "label1"));
-        assertEquals("label1", dynamicFixedListType.getSelectedLabel());
+        assertThat("label1", is(equalTo(dynamicFixedListType.getSelectedLabel())));
```

Comment:
Minor: This is good as using assertThat helps on readability.  Worth taking it one step further and using the assertThat which format's like this instead:
```
assertThat(dynamicFixedListType.getSelectedLabel())
   .isEqualTo("label1");
```

It puts it the right way round and simplifies a wee bit more.

Same applies to the rest of the assertThat's too as this alternative way (Hamcrest I believe) is cleaner for those scenarios too

Reply:
Yes that is the right order with assertThat, the opposite of assertEquals.  I sort it in a new pr.